var/spack/repos/builtin/packages/visit/package.py

```diff
@@ -58,6 +58,8 @@ class Visit(CMakePackage):
     executables = ["^visit$"]
 
     version("develop", branch="develop")
+    version("3.4.1", sha256="942108cb294f4c9584a1628225b0be39c114c7e9e01805fb335d9c0b507689f5")
```

Comment:
I am digging into this in more detail here:

https://github.com/spack/spack/pull/47161